common/djangoapps/util/milestones_helpers.py

```diff
@@ -86,33 +87,46 @@ def set_prerequisite_courses(course_key, prerequisite_course_keys):
             add_prerequisite_course(course_key, prerequisite_course_key)
 
 
-def get_pre_requisite_courses_not_completed(user, enrolled_courses):
+def get_pre_requisite_courses_not_completed(user, enrolled_courses):  # pylint: disable=invalid-name
```

Comment:
If this is easy to do, suggested name change:

get_unfulfilled_prerequisite_milestones or something of the sort. The terminology for milestones that is usually used is unfulfilled/fulfilled.